### ISSUE TITLE
Custom URL like "remote.mesh:888\laptopserialnumber"

### ISSUE THREAD

**Issue by UnP1u9** [Ylianst/MeshCentral#349]
can we make some arrangement so that we get a URL like "remote.mesh:888\laptopserialnumber" 

so that i can integrate it with my asset management tool where it show a icon in front of asset which brings me to the above link (by taking laptopserialnumber from my asset tool) and ask for creds and take remote access.

and also any way to integrate password with any different app like my asset management so that i can authenticate it from there. , node is lil new to me 😅

_Originally posted by @UnP1u9 in https://github.com/Ylianst/MeshCentral/issues/339#issuecomment-508725373_

**Comment by Ylianst** [comment]
Agreed. I do want to make sure we grab the same serial number. I will look at the SMBIOS tables I have now, I may already have this information. If I do, should not be difficult.

**Comment by Ylianst** [comment]
Like a stated in #339, the serial number as obtained by "WMIC BIOS GET SERIALNUMBER" is often not set and so, it's not reliable. If you have another asset tracking identifier, let me know.

**Comment by UnP1u9** [comment]
only laptop S/N is unique which can be used to track so nothing else for now, logged in Username is kind of unique but can be misspelled or multiple logins can be find on kiosk and others

we have around 500 laptop and all works fine with WMIC , will check if anything else it there to get same

**Comment by Ylianst** [comment]
Ok, I will try to find a computer that works with this so I can do some testing...
